packages/i18n/src/ja.ts

```diff
@@ -0,0 +1,66 @@
+import { Language } from './types';
+
+// prettier-ignore
+const language: Language = {
+  code:             'ja',
+  schema:           (issue) => `無効な型: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+  specific: {
+    bic:            (issue) => `無効なBIC: ${issue.received} を受け取った`,
+    bytes:          (issue) => `無効なバイト数: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+    creditCard:     (issue) => `無効なクレジットカード: ${issue.received} を受け取った`,
+    cuid2:          (issue) => `無効なCuid2: ${issue.received} を受け取った`,
+    custom:         (issue) => `無効な入力: ${issue.received} を受け取った`,
+    customAsync:    (issue) => `無効な入力: ${issue.received} を受け取った`,
+    decimal:        (issue) => `無効な10進数: ${issue.received} を受け取った`,
+    email:          (issue) => `無効なメールアドレス: ${issue.received} を受け取った`,
+    emoji:          (issue) => `無効な絵文字: ${issue.received} を受け取った`,
+    endsWith:       (issue) => `無効な末尾: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+    equal:          (issue) => `無効な値: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+    excludes:       (issue) => `無効な内容: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
+    finite:         (issue) => `無効な有限数: ${issue.received} を受け取った`,
+    hash:           (issue) => `無効なハッシュ値: ${issue.received} を受け取った`,
+    hexColor:       (issue) => `無効な16進数カラーコード: ${issue.received} を受け取った`,
+    hexadecimal:    (issue) => `無効な16進数: ${issue.received} を受け取った`,
+    imei:           (issue) => `無効なIMEI: ${issue.received} を受け取った`,
+    includes:       (issue) => `無効な内容: ${issue.expected} を期待したが、 ${issue.received} を受け取った`,
```

Comment:
```suggestion
    includes:       (issue) => `無効な内容: ${issue.expected} を以外の値を期待しましたが、 ${issue.received} を受け取りました`,
```

I added the words `以外の値`("value other than") to make it clear that not including a value is an error. However, when I look at other languages, it is not written in such detail, so I am confused about whether to express it in that way only in Japanese.

Reply:
This is why not do it a same reason as https://github.com/fabian-hiller/valibot/pull/431#discussion_r1485103638.